connect example doesn't compile with the lastest btcd wire

When trying to build the connect example two build errors are thrown.

```console
./peerHandler.go:45:14: undefined: wire.NewMsgVersionFromConn
./util.go:32:15: cannot assign 1 values to 2 variables
```

The underlying btcd wire library has changed.

--- a/example/connect.go
+++ b/example/connect.go
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"log"
 
-	"github.com/jonasnick/btcP2P"
+	".." 		// TODO: change back to "github.com/jonasnick/btcP2P" once merged
 )
 
 /*
@@ -39,7 +39,7 @@ func main() {
 	stateResultChan := make(chan *btcP2P.PeerState)
 	peer = btcP2P.NewPeer(p2pConn, stateResultChan)
 	go peer.BasicHandler()
-	go peer.NegotiateVersionHandler(ownIpAddress)
+	go peer.NegotiateVersionHandler(ownIpAddress, nodeAddress)
 	printResult(stateResultChan)
 	go peer.AddrRequest()
 	printResult(stateResultChan)

--- a/peerHandler.go
+++ b/peerHandler.go
@@ -41,24 +41,29 @@ OuterLoop:
 	peer.SetState(&StateDisconnected{})
 }
 
-func MyNewMsgVersionFromConn(conn net.Conn, ownIpAddress string) (*wire.MsgVersion, error) {
-	msg, err := wire.NewMsgVersionFromConn(conn, nonce, 332753)
-	if err != nil {
-		return nil, err
-	}
-	msg.Services = 1
-
+func MyNewMsgVersion(conn net.Conn, ownIpAddress string, nodeAddress string) (*wire.MsgVersion, error) {
 	addrMe, err := StrIpAddrToNetAddr(ownIpAddress)
 	if err != nil {
 		return nil, err
 	}
 
+	addrYou, err := StrIpAddrToNetAddr(nodeAddress)
+	if err != nil {
+		return nil, err
+	}
+
+	msg := wire.NewMsgVersion(addrMe, addrYou, nonce, 500000)
+	if err != nil {
+		return nil, err
+	}
+	msg.Services = 1
+
 	msg.AddrMe = *addrMe
 	return msg, nil
 }
 
-func (peer *Peer) NegotiateVersionHandler(ownIpAddress string) {
-	msg, err := MyNewMsgVersionFromConn(peer.conn, ownIpAddress)
+func (peer *Peer) NegotiateVersionHandler(ownIpAddress string, nodeAddress string) {
+	msg, err := MyNewMsgVersion(peer.conn, ownIpAddress, nodeAddress)
 	CheckError(err)
 	var otherVersion *wire.MsgVersion
 	otherVerack := false

--- a/util.go
+++ b/util.go
@@ -29,10 +29,7 @@ func StrIpAddrToNetAddr(ipAddress string) (*wire.NetAddress, error) {
 	if err != nil {
 		return nil, err
 	}
-	netAddr, err := wire.NewNetAddress(parsedAddr, 1)
-	if err != nil {
-		return nil, err
-	}
+	netAddr := wire.NewNetAddress(parsedAddr, 1)
 	return netAddr, err
 }
 

